src/buildercore/trop.py

```diff
@@ -377,6 +381,24 @@ def external_dns_cloudfront(context):
 
     return dns_records
 
+def external_dns_fastly(context):
+    "additional CNAME DNS entries pointing to a Fastly CDN"
+    assert isinstance(context['domain'], str), "A 'domain' must be specified for CNAMEs to be built"
+
+    def entry(hostname, i):
+        if _is_domain_2nd_level(hostname):
+            raise ConfigurationError("2nd-level domains aliases are not supported yet by builder. See https://docs.fastly.com/guides/basic-configuration/using-fastly-with-apex-domains")
+        hostedzone = context['domain'] + "."
+        return route53.RecordSetType(
+            R53_FASTLY_TITLE % (i + 1),
```

Comment:
expecting that there can be more than one entry (aliases), so numbering them immediately